docs/resources/validator-faq.md

```diff
@@ -98,10 +98,9 @@ Each validator receives revenue in proportion to its total stake. However, befor
 
 ###  IRISnet Valdiators Risks
 
--   **Unavailability**: Validators are expected to keep signing votes for making new blocks. If a validator’s signature has not been included in the last 20,000 blocks, 
-the validator will get slashed by 0.5% and remove from current validatorset for 2 days.
--   **Double Sign**：If someone reports on IRIShub that a validator signed two different blocks at the same height, this validator will get slashed on IRIShub by 1% and remove from current validatorset for 5 day.
--   **Censorship**：If someone reports that a validator includes invalid transactions in block, this validator will get slashed on IRIShub by 2% and remove from current validatorset for 7 day.
+-   **Unavailability**: Validators are expected to keep signing votes for making new blocks. If a validator’s signature has not been included in more than  half of the last 34,560 blocks (which amounts to approximately 24 hours, assuming an average block-generating time of 5 seconds)
```

Comment:
... 5 seconds), this validator will get jailed and removed from current ValidatorSet for 1 day.